### ISSUE TITLE
Not really a save editor, more a bad text editor

### ISSUE THREAD

**Issue by BingusManLeftToenail** [CodexOne03/Clone-Drone-Save-Editor#1]
Its just notepad with extra steps..? I could accomplish the same thing this is trying to do with VS-Code or Notepad++, I'd recommend making the GUI easier to read or maybe something that allows you to actually edit and see the values easier or understand what they do. Save editors generally do more than just a text editor.

**Comment by CodexOne03** [comment]
Yeah, sorry about that. This is a really old project, and it's written really poorly. I'll rewrite it as soon as possible. Hope to hear your feedback again in the future.
